solarkraft/src/fetcher/storage.ts

```diff
@@ -128,6 +144,41 @@ export function loadContractCallEntry(filename: string): ContractCallEntry {
     }
 }
 
+/**
+ * Generate storage entries for a given contract id in a path.
+ * @param contractId contract identifier (address)
+ * @param path the path to the contract storage
+ */
+export function* yieldListEntriesForContract(
+    contractId: string,
+    path: string
+): Generator<ListEntry> {
+    for (const dirent of readdirSync(path, { withFileTypes: true })) {
+        if (dirent.isDirectory() && /^[0-9]+$/.exec(dirent.name)) {
+            // This directory may contain several transactions for the same height.
+            const height = Number.parseInt(dirent.name)
+            for (const ledgerDirent of readdirSync(join(path, dirent.name), {
+                withFileTypes: true,
+            })) {
+                // match all storage entries
+                const matcher = /^entry-([0-9a-fA-F]+)\.json$/.exec(
```

Comment:
Why are we using mixed-case HEX here?

Reply:
I would be ok with lowercase, but neither Windows, nor MacOS agree with me on the capitalization of filenames

Reply:
added a comment